session/src/test/java/uk/gov/di/ipv/cri/common/api/service/SessionRequestServiceTest.java

```diff
@@ -157,40 +144,30 @@ void shouldFailOnInvalidDOBInClaims() throws Exception {
                         mockJwtDecrypter);
 
         JSONObject requestBody = new JSONObject();
-        requestBody.put("client_id", "client-id");
-        requestBody.put("requestJWT", "request");
+        requestBody.put("client_id", "clientSessionIdTest");
+        requestBody.put(
+                "request",
+                "eyJjdHkiOiJKV1QiLCJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiUlNBLU9BRVAtMjU2In0.H2nSDy6ovr-GSHLFWroLUDUDID7SMjbAmtZMENv6DVFtP679d3jrowTiVqWdhZImP_ADF_tWVbwBcFsCVBIuFektqdHl6h5kKr90jBu4gPO_OMXcJTtjVaurBwCHRTlMb_7QYQZlh53I2NNQB3VIWz39NStIHo9_1l-AgEvzeYwpJFHd36empaeeYRS7qds-xStWh3H38mIibXaaPwwkQlDEnP2XNsaqPcxhWtJqFJ4lE6JgBxvqxEeazWgsZYZI4p97UXTMkbIS3ECKvnpDXn8fyGLSX1qYxYosLaQZBUAgpCg2DCw-cnYeToR_fl8jNas762KcEJyO8DsNr9joNA.Mq1fLGkQL8EBosZ8.U0qDIWuZv3p0eZ7-nFto4Vnj1IMb5CnYccAbAqCFPHrSJzuyzWAMBBDKi1LE47KZZxL9nn8r1iY86lOrEk5eVgFGEl-vd5KzZF34ccvgajQLnLi0uuxi8JP3d6z_Wq2cWAuqfJsuBtEDvqLUwVapHXqhksTNK40lmW-VMhNP6YfJILyI8L0U9896HCt4Q6MlhC0OA5-CWYLlB3_7dHDqG0qiiCKQBLE3avtnEblDrSlO5abBx2Iq2xmcuP7FivzH_1AUrBwU3zsqa08-AMFOTdvOEUJEpwHAjM9PKZHjtImvau91IV0cbb5B-8cI_BO4FZVCFU26tF0dk4f1rn_ZcE6a4YR1_RlcnerqevRO5AIOY6HTWal18Jb6Djr3mfsPvFoId9Ip0eK4X8b5g_T_jukEV61tXoDkKiwlpk9WiR0PPlQAoqZm-N9Ut5_vht1D-2ZRwKnho1aDmR_GoaOd1kKR1TUA4uTRsldAbnR0jOUrgxO2AO9GMT--LtK4ZqgBU8jBfKJy3-AeW8adOK5q2HBFQl1BD3Mt2Ptucu14FjgedGLlPromOLWCwi9Nq1gcPt111EBpA_Z3tBVFUmmWoC17aFU06-QwPG1n71i6ZTFLDtYPNZTWWAyrnUILa0GiOqw5AFddfs_zpf7rcTQgWRHOLkp4kU1ZCbORx4HYg1SVdJWIuKZZjcC1UpavBAk8IDg8yJog0W4fs8ZdqREog7XAjiPyJBUgGmh_BaS1qPlQS0DS2j-Qx7DJ628etHZlTS2wZbFrsKr8C2o1E0dDO4T2Atl2DSLwyvGO_NECV-90ZdDBf5QtHA43vOi4YwaASn1mVaP6g1_sVMZzjxHmmUtguM7P2UyqvGN-IJ5F5H65tdxbCLf9fydnQT5-Gt8-QsvGa6gYiQwJRreTcD3_BaoJrfJhWydYOnegmH_8DZd01qDMNDBXo9lWBWjODMF1WysryojFv2wW3OxiCuWoG7psV4zuctiTAryLjEAlA_25eTTUS-GOaapIrbNH8RBzm5a3Gttww5tMlISHg3nSwCWxtzslVXzDqXOk6g2hecr2dw1LjSV0YD4Z5pW_3LnYfQjMpoIBIPL3YL4t8tEW3koNInlas4hN--E7AUOv3_1_IqYGWsxyFLiPSms5_r9ez8hD_bvUDOOXQbF8NK_TdMVvQX80Ywkfj2RPD6B8Xdf0cWVL0fFatYchWEz5jVzIlT1bJB7S1h8WKKtOHMBKulXni8div2wYCEUqOHR3LXJRIuzFA6-DDvQv44k_dYuO0gXHGzC-1M1AwDXUQ2tvgcmB44hdzIaWJ88nbh3WN4Xr2MTYSoOgMpIAbduZ1QZXR8REMCWTjp3BCRKI3-PQmevdqgxbMTswrniJXmoDqaIJYYvX_7Sq1qCY4cfKXr4mESkr68VyCSAEazOk-XUyaS-1UnGBjQtp7W9kpZFRpX-mgcJczD2HIhgGNmT3_1Hh9LjPnE2j2ZUfC4zeZG2tIYN-IUujS2N5loyOm0a6F_0p3MqWmuLBzCEm6rsAYxaVtpWbJlwASbF9KpdxXe67QyR27H4JjYucei9HWfqR44hKWzaBMMVrVxgkYhJRVhesQgZNP9C5C9vrZPHkd7GlCGkDB5Yn-7T2pU7fSasfF6K0o5vHIocF3RTypy0oDdRwulHu-8Ih.GCTlL_UWJcCiE0L3khUjvQ");
```

Comment:
Using the encrypted JWT makes the test more fragile to change as if we need to change something (e.g. functionality changes) it'll be difficult to maintain this.